content/news/fpc-24-registration.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Freshmen Programming Contest 2024"
+date: 2024-03-13T12:00:00+02:00
+tags: ["FPC 2024"]
+draft: true
+---
+Are you a freshmen and do you want to solve puzzles in a team with your friends?
+Then the Freshmen Programming Contest is for you!
+
+On the 4th of May, the annual Freshmen Programming Contest will be held.
+The puzzles are suited for both beginners and experts, so don't be afraid to join!
+This contest is a great opportunity for you to sharpen your programming and puzzling skills!
+
+The event lasts approximately from 10:30 until 17:30. More details will be sent
+when you register. Teams with non-freshmen members are able to join but will be
+assigned to the spectator pool, meaning they do not count towards the final ranking.
```

Comment:
>Teams with non-freshmen members are able to join but will be assigned to the spectator pool, ...

Maybe add "as space allows," after "able to join", to cover ourselves for when too many spectator teams want to join?